### ISSUE TITLE
Accessibility issue: thumbnails in gallery view not keyboard-accessible

### ISSUE THREAD

**Issue by scoutb-cogapp** [UniversalViewer/universalviewer#1067]
**UV version:** _universalviewer@4.0.25_

**I'm submitting a:** *bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below*

### Page area
thumbnails in gallery view

### Issue description
The thumbnails in the gallery view cannot be accessed by keyboard, they are just skipped when tabbing.

### Steps to reproduce
1. open [this manifest](https://www.universalviewer.dev/#?xywh=583%2C1062%2C2492%2C1379&iiifManifestId=https%3A%2F%2Feap.bl.uk%2Farchive-file%2FEAP500-1-1%2Fmanifest&cv=)
2. activate Icon gallery button in top right corner to trigger the gallery view
3. press tab key and observe focus moving into the gallery view: it goes to minus button, slider, plus button, and then moves outside the gallery view skipping all the thumbnails

### Expected behaviour
keyboard users are able to tab to each item, including the thumbnails, so they are able to navigate and activate these thumbnails like users with a mouse

### WCAG criterion
2.1.1 Keyboard (Level A)

### Related code

```html
<div class="wrap loaded" style="width: 120px; height: 150px;" data-link="class{merge:multiSelected toggle='multiSelected'}"><img class="thumbImage" src="http://images.eap.bl.uk/EAP500/EAP500_1_1/1.jp2/full/200,/0/default.jpg" style="">						<script type="jsv#1^"></script><script type="jsv/1^"></script>						</div>
</div>
```

**Comment by demiankatz** [comment]
This is an issue with https://github.com/IIIF-Commons/iiif-gallery-component rather than the UV itself -- a fix will have to be applied to that project and a new release of the component made.

**Comment by demiankatz** [comment]
See https://github.com/IIIF-Commons/iiif-gallery-component/pull/20 for work in progress.
